app/controllers/miq_policy_controller/alert_profiles.rb

```diff
@@ -149,8 +149,8 @@ def alert_profile_field_changed
     return unless load_edit("alert_profile_edit__#{params[:id]}", "replace_cell__explorer")
     @alert_profile = @edit[:alert_profile_id] ? MiqAlertSet.find(@edit[:alert_profile_id]) : MiqAlertSet.new
 
-    @edit[:new][:description] = params[:description].blank? ? nil : params[:description] if params[:description]
-    @edit[:new][:notes] = params[:notes].blank? ? nil : params[:notes] if params[:notes]
+    @edit[:new][:description] = params[:description].presence if params[:description]
+    @edit[:new][:notes] = params[:notes].presence if params[:notes]
```

Comment:
This is a little weird, is it not allowed to have `nil` values here?

Reply:
Don't know. With the previous code, `@edit[:new][:description]` and `@edit[:new][:notes]` would never be `nil`, correct?.

Reply:
Anyway, this should be fixed.